scripts/mail-alarm

```diff
@@ -676,60 +661,68 @@ class XapiMessage:
 
 def main():
     if len(sys.argv) < 2:
-        log_err("Expected at least 1 argument but got none: [\"%s\"]." % (' '.join(sys.argv)))
-        raise Exception("Insufficient arguments")
+            log_err("Expected at least 1 argument but got none: [\"%s\"]." % (' '.join(sys.argv)))
```

Comment:
Indentation looks wrong. Please run all of this trough some automation indention tool